### PR TITLE
👩‍🌾 1 ➡️ 2

### PR DESCRIPTION
**Pull request by chapulina**
# ➡️ Forward port

Port `ign-tools1` to `main`

Branch comparison: https://github.com/ignitionrobotics/ign-tools/compare/main...ign-tools1

**Note to maintainers**: Remember to **Merge** with commit (not squash-merge or rebase)

---

https://github.com/osrf/buildfarmer/issues/161

### CODE REVIEW

**Comment by chapulina** [comment]
The `Ubuntu Bionic CI` failures are expected, this is the CI that's being removed. I've already unchecked the "required" box for them but I guess GitHub will take a while to update them.

**Comment by azeey** [comment]
The `ci-bionic.yml` workflow file is still there. Did you mean to remove it?

**Review by adlarkin — COMMENTED** [review]
Other than @azeey's comment, everything here looks okay to me. The history in the branch comparison was a little hard to parse at times (it looks like some things weren't squash-merged), so if someone else wants to look this over as a sanity check, that would be great!

**Comment by chapulina** [comment]
> The ci-bionic.yml workflow file is still there. Did you mean to remove it?

Nice catch, removed

> Ubuntu Bionic CI failures are expected, this is the CI that's being removed. I've already unchecked the "required" box for them but I guess GitHub will take a while to update them.

It looks like they're finally gone

> The history in the branch comparison was a little hard to parse at times (it looks like some things weren't squash-merged)

Yeah that's still some BitBucket / mercurial legacy

**Review by adlarkin — APPROVED** [review]

**Comment by adlarkin** [comment]
It looks like windows CI failed, but I can't figure out why when I look in Jenkins.

**Comment by chapulina** [comment]
> It looks like windows CI failed, but I can't figure out why when I look in Jenkins.

Whenever I see a lot of `at hudson.` messages, I assume it was an infrastructure failure. I disabled that Jenkins node and retriggered a build.

CC :man_farmer: @azeey
